lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -574,14 +574,59 @@ ADDRESS_AND_FRAUD_J7:
       targetState: CRI_HMRC_KBV_J6
```

Comment:
Do we also need to update the HMRC KBV route as well - what if you get a V03 there? (Or is that a separate ticket?)